xds/internal/xdsclient/xdsresource/unmarshal_cds_test.go

```diff
@@ -1347,6 +1347,23 @@ func (s) TestUnmarshalCluster(t *testing.T) {
 				Raw: v3ClusterAnyWithEDSConfigSourceSelf,
 			},
 		},
+		{
+			name: "xdstp cluster resource with unset EDS service name",
+			resource: testutils.MarshalAny(&v3clusterpb.Cluster{
+				Name:                 "xdstp:foo",
```

Comment:
Should this be `xdstp:///foo`? I understand it doesn't matter for the purpose of this unit test. But was just curious if `xdstp:foo` was a valid name at all.